src/lib/netlist/devices/nlid_system.h

```diff
@@ -287,8 +288,9 @@ namespace netlist::devices {
 		, m_compiled(*this, "m_compiled")
 		, m_last(*this, "m_last")
 		{
+			m_I.reserve(m_N());
 			std::vector<pstring> inputs;
-			for (int i=0; i < m_N(); i++)
+			for (uint64_t i=0; i < m_N(); i++)
```

Comment:
Same here – you’ve got two more vectors that you can preallocate to make the loop cheaper.

Reply:
Good catch. Done. Also fixed VS build.